includes/commands/class-mu-migration-base.php

```diff
@@ -8,14 +11,14 @@
 	/**
 	 * Holds the command arguments
 	 *
-	 * @var Array
+	 * @var array
```

Comment:
Right, will leave them then.

Also, I'll need to add a lot of `.`s at the end of the summaries and descriptions.

(Since we're at it might as well [go all the way](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods))